services/datamanager/builtin/builtin.go

```diff
@@ -216,20 +220,26 @@ func getDurationFromHz(captureFrequencyHz float32) time.Duration {
 // Return the component/method metadata which is used as a key in the collectors map.
 func (svc *builtIn) initializeOrUpdateCollector(
 	md componentMethodMetadata,
-	config dataCaptureConfig) (
+	config *datamanager.DataCaptureConfig,
+) (
 	*collectorAndConfig, error,
 ) {
 	// Build metadata.
-	captureMetadata, err := datacapture.BuildCaptureMetadata(config.Type, config.Name,
-		config.Model, config.Method, config.AdditionalParams, config.Tags)
+	captureMetadata, err := datacapture.BuildCaptureMetadata(
+		config.Name.Subtype,
+		config.Name.ShortName(),
```

Comment:
Q: why ShortName vs Name? Is this necessary for supporting remotes it looks like?